typings/internal-electron.d.ts

```diff
@@ -136,6 +136,7 @@ declare namespace Electron {
     overrideReadOnlyProperty(property: string, value: any): void;
     groupId: number;
     getDefaultRoleAccelerator(): Accelerator | undefined;
+    getCheckStatus(): boolean;
```

Comment:
I think `getCheckStatus()` can return `undefined`?

Reply:
The `roles.getCheckStatus()` can return `undefined`, but `MenuItem.getCheckStatus` can not, the code path that returns `undefined` should have been avoided.